config-linux.md

```diff
@@ -545,6 +545,16 @@ Its value is either slave, private, or shared.
     ]
 ```
 
+## Mount Label
+
+`mountLabel` will set the Selinux context for the mounts in the container.
+
+###### Example
+
+```json
+    "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
```

Comment:
sorry. One more nit. Please add this line to the "complete json" example in the config.md.